src-java/server42/server42-control-storm-topology/src/main/java/org/openkilda/server42/control/topology/service/RouterService.java

```diff
@@ -52,4 +58,29 @@ public void handleFlowRttFeatureToggle(Boolean featureEnabled) {
                     .forEach(s -> carrier.deactivateFlowMonitoringOnSwitch(s.getSwitchId()));
         }
     }
+
+    /**
+     * Part of LCM. Sends flow sync messages to FlowHandler or deactivate in case of feature disabled.
+     */
+    public void processSync() {
+        if (isFlowRttFeatureToggle()) {
+            Collection<SwitchProperties> all = switchPropertiesRepository.findAll();
+            all.stream()
```

Comment:
As for me simple loop will be better that filter stream 2 times.

Reply:
It's a matter of taste.